packages/protocol-contracts/contracts/ZetaConnector.base.sol

```diff
@@ -47,39 +47,42 @@ contract ZetaConnectorBase is Pausable, ZetaConnectorErrors {
     event TSSAddressUpdated(address originSenderAddress, address newTSSAddress);
 
     constructor(
-        address zetaTokenAddress_,
+        address zetaTokenAddress,
         address tssAddress_,
         address tssAddressUpdater_
     ) {
-        if (zetaTokenAddress_ == address(0) || tssAddress_ == address(0) || tssAddressUpdater_ == address(0))
-            revert AddressCantBeZero();
-        zetaToken = zetaTokenAddress_;
+        if (zetaTokenAddress == address(0) || tssAddress_ == address(0) || tssAddressUpdater_ == address(0)) {
+            revert InvalidAddress();
+        }
+
+        zetaToken = zetaTokenAddress;
```

Comment:
it's more consistent if the param it's called zetaToken_ like the others

Reply:
Strongly agree